fedimintd/src/fedimintd.rs

```diff
@@ -367,14 +360,8 @@ impl Fedimintd {
                     sleep(SHUTDOWN_TIMEOUT).await;
```

Comment:
aren't we always sleeping for 10s herre

Reply:
I don't understand what happened. I'm quite sure I was not seeing it.

But anyway - I got rid of this now, and also nested api runtime shenanigans that were causing a panic.